hazelcast/src/test/java/com/hazelcast/instance/impl/HazelcastInstanceFactoryTest.java

```diff
@@ -183,7 +183,7 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
                         }
                         return null;
                     }
-                }).when(nodeExtension).beforeShutdown(false);
+                }).when(nodeExtension).beforeShutdown(true);
```

Comment:
Maybe the Mockito's `ArgumentMatchers.anyBoolean()` would be a safer choice for the argument there